maintainer/channels.py

```diff
@@ -1,24 +1,20 @@
+import logging
 import requests
 import configuration
-
+from common import from_days_to_seconds
 
```

Comment:
Is this spacer pep8 compatible? Was it linted?

Reply:
fixed